internal/assets/contents/com.activestate.platform.app.plist.tpl

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>CFBundleExecutable</key>
+        <string>{{.Exec}}</string>
+        <key>CFBundleIconFile</key>
+        <string>{{.Icon}}</string>
+        <key>CFBundleIdentifier</key>
+        <string>com.activestate.platform.{{.Exec}}</string>
+        <key>NSHighResolutionCapable</key>
+        <true/>
+        <key>LSUIElement</key>
+        <true/>
```

Comment:
These seem irrelevant for CLI applications. Maybe we need to have some sort of setting for the App package that says "this is a UI app" vs "this is a CLI app".

Reply:
We will want to keep `LSUIElement` as that denotes that the app runs in the background and there isn't an icon in the doc. Added an option for that as well.